test/controllers/api/v2/config_reports_controller_test.rb

```diff
@@ -48,9 +48,10 @@ def test_create_duplicate
       Setting[:require_ssl_smart_proxies] = false
 
       proxy = smart_proxies(:puppetmaster)
+      proxy.update_attribute(:url, 'http://configreports.foreman')
```

Comment:
Yeah, without it `auth_smart_proxies` in the controller will find other proxies with the feature that have 'else.where' in the name, and those might not necessarily have the Puppet feature